types/tablestore/index.d.ts

```diff
@@ -0,0 +1,645 @@
+// Type definitions for tablestore 5.1
+// Project: https://github.com/aliyun/aliyun-tablestore-nodejs-sdk
+// Definitions by: craigwu <https://github.com/waynecraig>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+// ---------- metadata ----------
+export enum rowExistenceExpectation {
+    IGNORE,
+    EXPECT_EXIST,
+    EXPECT_NOT_EXIST,
+}
+
```

Comment:
Isn't that one a duplicate? `RowExistenceExpectation`

Reply:
Both TableStore.RowExistenceExpectation and TableStore.rowExistenceExpectation exist in the source code of tablestore. They have same keys but different values, so it's not a duplicate. I searched all documentation and examples, didn't find any use case for rowExistenceExpectation. I thought rowExistenceExpectation was useless, so I removed it in the following commit to avoid confusion.